lib/setup.js

```diff
@@ -229,14 +229,6 @@ export default function (app) {
     app.use(require('../desktop'))
   }
 
-  // Direct mobile devices to the mobile app, otherwise fall through to
-  // the desktop app
-  app.use((req, res, next) => {
-    if (res.locals.sd.IS_MOBILE) require('../mobile')(req, res, next)
-    else next()
-  })
-  app.use(require('../desktop'))
```

Comment:
Noticed that this bit of code was erroniously being duplicated and so I removed it.